config/analytics.yml

```diff
@@ -101,6 +101,7 @@
     - teaching_authority_emails
     - teaching_authority_websites
     - teaching_authority_name
+    - teaching_authority_other
```

Comment:
Do we need to update data form for this field? I wouldn't mind doing that with you if so as I've not done any of that before.

Reply:
Yes, we will need to change this in dataform. I'll put something in the calendar to pair on it!